scripts/rename_resources.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os.path
+import subprocess
+
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+TERRAFORM_PATH = 'terraform'
```

Comment:
We don't use upper case for pseudo constants. COuld you add a rule to the C guide?

Besides, it's fine to inline this altogether. There is only one reference. I don't believe in trying to anticipate anything that might vary in the future and extract it into a pseudo-constant.

Reply:
> We don't use upper case for pseudo constants. COuld you add a rule to the C guide?

#2128

> Besides, it's fine to inline this altogether. There is only one reference. I don't believe in trying to anticipate anything that might vary in the future and extract it into a pseudo-constant.

I used this form to avoid a `Name 'TERRAFORM_PATH' can be undefined` warning if it is defined only inside the `try:` block. In cases like this, is there a better way to write it?